lib/typeform_data/client.rb

```diff
@@ -6,13 +6,13 @@ class Client
 
     # For the sake of usability, we're breaking convention here and accepting an API key as the
     # first parameter instead of an instance of TypeformData::Config.
-    def initialize(api_key:)
-      @config = TypeformData::Config.new(api_key: api_key)
+    def initialize(api_key:, logger: nil)
```

Comment:
Maybe it goes without saying, but it would be helpful to call out here that we expect the logger to implement the https://ruby-doc.org/stdlib-2.1.0/libdoc/logger/rdoc/Logger.html API.